erizo/src/erizo/rtp/RtcpNackGenerator.cpp

```diff
@@ -89,8 +89,7 @@ bool RtcpNackGenerator::addNackPacketToRr(std::shared_ptr<DataPacket> rr_packet)
     base_nack_info.sent_time = now_ms;
     base_nack_info.retransmits++;
     while (index + 1u < nack_info_list_.size()) {
-      index++;
-      NackInfo& blp_nack_info = nack_info_list_[index];
+      NackInfo& blp_nack_info = nack_info_list_[index + 1];
       uint16_t distance = blp_nack_info.seq_num - pid -1;
       if (distance <= 15) {
         if (!isTimeToRetransmit(blp_nack_info, now_ms)) {
```

Comment:
We would need another `index++` here. Otherwise we'd be stuck testing for the same nack_info until the time is right to send it.

Reply:
Didn't see that at all, thanks!